firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -244,7 +249,7 @@ private FirebaseFirestoreSettings mergeEmulatorSettings(
 
     if (!FirebaseFirestoreSettings.DEFAULT_HOST.equals(settings.getHost())) {
       throw new IllegalStateException(
-          "Cannot specify the host in FirebaseFirestoreSettings when EmulatedServiceSettings is provided.");
+          "Cannot specify the host in FirebaseFirestoreSettings when emulator settings are provided.");
```

Comment:
This is allowed in the RTDB proposal. No strong opinion here, but just wanted to point this out.

Reply:
So would you prefer the Firestore settings override?

Reply:
I do, but I defer to your judgement.